Packs/Darktrace/Integrations/DarktraceEventCollector/DarktraceEventCollector.yml

```
@@ -0,0 +1,102 @@
+category: Network Security
```

```suggestion
category: Analytics & SIEM
```
I verified this with @meital-aizen today, sorry for the confusion.